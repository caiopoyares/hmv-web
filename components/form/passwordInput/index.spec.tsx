import { render } from "@testing-library/react";
import { PasswordInput } from ".";

describe("PasswordInput", () => {
  it("should render default", () => {
    const { container } = render(
      <PasswordInput
        placeholder="senha"
        register={() => null}
        isInvalid={false}
        isRequired
      />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});

export {};
