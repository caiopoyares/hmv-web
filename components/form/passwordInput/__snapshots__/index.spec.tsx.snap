// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordInput should render default 1`] = `
<div
  class="chakra-input__group css-4302v8"
>
  <input
    class="chakra-input css-1n0hj84"
    placeholder="senha"
    test-id="password-input"
    type="password"
  />
  <div
    class="chakra-input__right-element css-7xoowg"
  >
    <button
      class="chakra-button css-13fla2j"
      test-id="show-btn"
      type="button"
    >
      Ver
    </button>
  </div>
</div>
`;
